pkg/kubelet/cm/cgroup_manager.go

```diff
@@ -0,0 +1,104 @@
+// +build linux
+
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cm
+
+import (
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/fs"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+// libcontainerCgroupManager implements CgroupManager interface.
+// It Uses the Libcontainer raw fs cgroup manager for cgroup management.
+type libcontainerCgroupManager struct {
+	// Libcontainer raw fs cgroup manager.
+	fsCgroupManager cgroups.Manager
+}
+
+// Make sure that libcontainerCgroupManager implements the CgroupManager interface
+var _ CgroupManager = &libcontainerCgroupManager{}
+
+// Returns libcontainer's cgroups.config{} struct given the general cgroupConfig
```

Comment:
getLibcontainerCgroupConfig returns libcontainer's cgroups.config{} struct given the general cgroupConfig.
See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences